examples/hosts/app-integration/external-data/src/view/taskListView.tsx

```diff
@@ -103,13 +104,18 @@ const TaskRow: React.FC<ITaskRowProps> = (props: ITaskRowProps) => {
 					{externalDataSnapshot.priority}
 				</td>
 			)}
-			<td>
-				<button
-					onClick={task.overwriteWithExternalData}
-					style={{ visibility: showAcceptButton }}
-				>
-					Accept change
-				</button>
+			<td style={{ visibility: showAcceptButton }}>
```

Comment:
Hiding a `<td>` doesn't mess up the layout? I'd expect something weird to happen visually if certain rows of a table don't have the same number of columns as others.